diff --git a/lzstring/__init__.py b/lzstring/__init__.py
index a264411..f157e4a 100644
--- a/lzstring/__init__.py
+++ b/lzstring/__init__.py
@@ -8,8 +8,9 @@ from __future__ import absolute_import
 from builtins import range
 from builtins import int
 from builtins import chr
-from future import standard_library
-standard_library.install_aliases()
+# These lines break the Python 3.12 compatilibity. They were needed only needed for Python 2, so should be fair to simply remove.
+# PR to fix upstream: https://github.com/gkovacs/lz-string-python/pull/6
+# from future import standard_library
+# standard_library.install_aliases()
 from builtins import object
 import math
 import re
